geoips/commandline/ancillary_info/cmd_instructions.yaml

```diff
@@ -17,15 +17,15 @@ instructions:
       To use, type `geoips config install <test_dataset_name>`.
     output_info:
       - Not Applicable
-  geoips_get: &geoips-get-interface
+  geoips_describe: &geoips-describe-artifact
     help_str: |
       Retrieve information about a GeoIPS artifact. For this command, an artifact is one
-      of ['interface', 'plugin', 'family']. To get more information on how to retrieve
-      these artifacts, see below.
+      of ['interface', 'plugin', 'family']. To describe more information on how to
```

Comment:
"describe" seems inappropriate here.

Reply:
Ah, that was a mistake on my part. I was just doing a find and replace and accidentally included that 'get'. I'll change that back. Let me know if you find any more of those!

Reply:
I hope @mindyls has told you about "single_varnel".